docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst

```diff
@@ -104,21 +104,106 @@ Wasting and Stunting
+
+.. todo::
+
+   Link to x-factor documentation for wasting model when it is developed
+
 Correlation
 ^^^^^^^^^^^^
 
+Initialization
+~~~~~~~~~~~~~~~
+
+When simulants are initialized into the early or late neonatal age groups or born into the simulation, they will be assigned a birthweight exposure value. When they transition into the postneonatal age group, they should be initialized into a particular state in the wasting model (e.g. TMREL, mild wasting, moderate wasting, or severe wasting) *based on their birthweight percentile within their overall location- and sex-specific birthweight distribution among the late neonatal age group.*
+
+This should be done according to the methodology described in the :ref:`risk-risk correlation proposal page <2017_risk_models>`.
+
+.. todo::
+
+   Add more detail on the methodology here/as implemented in the BEP project
+
+.. note::
+
+   For simulants who are initialized into age groups older than the late neonatal age group, they will not be assigned a birthweight exposure, so they should be assigned a wasting exposure value that is independent of their birthweight.
+
+Transitions
+~~~~~~~~~~~~
+
+.. todo::
+
+   Add detail on how to correlate wasting x-factor propensity to birthweight exposure
+
 Causation
 ^^^^^^^^^^^
 
+For interventions that impact birthweight, there should be a corresponding change in a child's wasting risk exposure attributable *if the impact of the intervention on wasting is not modeled directly*.
+
+Initialization
+~~~~~~~~~~~~~~~~
+
+For each gram increase in a simulant's birthweight due to a simulation intervention, the category 1 (severe wasting) and category 2 (moderate wasting) exposures used to determine the probability of initialization into those states should be reduced proportionately such that the total reduction in moderate and severe wasting is equal to 0.0115 / 200 = 0.00575. The exposure distribution of category 3 (mild wasting) should be increased by 0.0115 / 200 = 0.00575. The figure below demonstrates how to implement this change visually.
+
```

Comment:
Do we apply this reduction in wasting from birth?

Reply:
Yes, added this in the latest commit!